docs/en/modules/admin/pages/features/notifications.adoc

```diff
@@ -15,3 +17,112 @@ image:features/notifications/notifications.png[Example of notifications page whe
+|                                          | Your initiative has achieved the X% of signatures                                     | ✅     | ❌          | ❌
+|                                          | The initiative has achieved the X% of signatures                                      | ❌     | ✅          | ❌
+| Accountability                           | The proposal has been included in a result                                            | ✅     | ❌          | ❌
+|                                          | The result progress has been updated                                                  | ✅     | ❌          | ❌
+| Blogs                                    | A post has been published                                                             | ✅     | ❌          | ❌
+| Budgets                                  | The budget is now active                                                              | ✅     | ❌          | ❌
+| Comments                                 | A ressource has been commented                                                        | ✅     | ❌          | ❌
+|                                          | A user group has left a comment on a ressource                                        | ✅     | ❌          | ❌
+|                                          | A user has left a comment on a ressource                                              | ✅     | ❌          | ❌
+|                                          | A user has replied your comment                                                       | ❌     | ✅          | ❌
+|                                          | A group you belong to has been mentioned                                              | ❌     | ✅          | ❌
+|                                          | You have been mentioned                                                               | ❌     | ✅          | ❌
+|                                          | Your comment in has been upvoted                                                      | ❌     | ✅          | ❌
+|                                          | Your comment in has been downvoted                                                    | ❌     | ✅          | ❌
+| Debates                                  | A debate has been created                                                             | ✅     | ❌          | ❌
+|                                          | Debate creation is enabled for participants                                           | ✅     | ❌          | ❌
+|                                          | Debate creation is no longer active                                                   | ✅     | ❌          | ❌
+|                                          | The debate was closed                                                                 | ✅     | ❌          | ❌
+|                                          | The debate was closed                                                                 | ❌     | ✅          | ❌
```

Comment:
It's not clear to me why there are two entries of the same thing here?

Reply:
Because in my original board, I was distinguishing between events, and there is one for affected_users, and one for followers, but I agree with you it's not clear at all!